src/flask_uploads/flask_uploads.py

```diff
@@ -148,14 +147,14 @@ def __init__(
             allow: Union[Tuple[()], Tuple[str, ...]] = (),
             deny: Union[Tuple[()], Tuple[str, ...]] = ()
     ) -> None:
-        self.destination = destination
+        self.destination = Path(destination)
```

Comment:
... so e.g. this conversion shouldn't be necessary as destination should already be a path object.